src/lib/crypto/kimchi/wasm/src/pasta_fp_plonk_index.rs

```diff
@@ -43,10 +43,8 @@ pub struct WasmPastaFpLookupTable {
 // below.
 impl From<WasmPastaFpLookupTable> for LookupTable<Fp> {
     fn from(wasm_lt: WasmPastaFpLookupTable) -> LookupTable<Fp> {
-        LookupTable {
-            id: wasm_lt.id.into(),
-            data: wasm_lt.data.0,
-        }
+        LookupTable::create(wasm_lt.id.into(), wasm_lt.data.0)
```

Comment:
Revert. This comes from a reverted and buggy PR. Not compatible with proof-systems normally.